infrastructure/main.tf

```diff
@@ -22,9 +27,14 @@ data "azurerm_key_vault" "fees_key_vault" {
   resource_group_name = "${var.product}-${local.local_env}"
 }
 
+data "azurerm_key_vault" "payment_key_vault" {
+  name = "${local.core_product_vaultName}"
+  resource_group_name = "ccpay-${var.env}"
+}
+
 data "azurerm_key_vault_secret" "appinsights_instrumentation_key" {
```

Comment:
you need to store the app insights key in your vault as well

Reply:
its should be ok now I have added = which was missing